backpack-lint/src/main/java/net/skyscanner/backpack/lint/IssueRegistry.kt

```diff
@@ -0,0 +1,34 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.lint
+
+import com.android.tools.lint.client.api.IssueRegistry
+import com.android.tools.lint.detector.api.CURRENT_API
+import com.android.tools.lint.detector.api.Issue
+import net.skyscanner.backpack.lint.check.BpkComponentUsageDetector
+
+@Suppress("unused", "UnstableApiUsage")
+class IssueRegistry : IssueRegistry() {
```

Comment:
we need to keep a list of our lint checks in a registry, so they get picked up by the system